data/json/items/tools.json

```diff
@@ -1,4 +1,4 @@
-[
+﻿[
```

Comment:
I solved the BOM issue, but I made a new one.
![2015-02-06_2230](https://cloud.githubusercontent.com/assets/5740367/6090673/e2cf91be-ae4f-11e4-8b11-c70d959d33e7.png)
I'm sure that's just an option that I need to enable. Mind letting me know which one?